DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -582,6 +593,21 @@ void printDigitHCHeader(o2::trd::DigitHCHeader& header, uint32_t headers[3])
   }
 }
 
+int getNumberOfTrackletsFromHeader(o2::trd::TrackletMCMHeader* header, bool verbose)
```

Comment:
Don't we already have this method? Duplicate?

Reply:
odd, and gone.